src/docs/patterns/visualizations.mdx

```diff
@@ -1,62 +1,93 @@
-import { DemoCard, ImageGrid, MaterialDesignDescription } from '../../app/components';
+import { DemoCard, ImageGrid, MaterialDesignDescription, Divider } from '../../app/components';
 
 import VisualizationBanner from '../../app/assets/design-patterns/visualizations/visualizations-banner.svg';
+import LineChart from '../../app/assets/design-patterns/visualizations/line-chart.png';
+import PieChart from '../../app/assets/design-patterns/visualizations/pie-chart.png';
+import ClockwisePie from '../../app/assets/design-patterns/visualizations/cw-ccw.png';
+import MapsImg from '../../app/assets/design-patterns/visualizations/map.png';
+import DontUsePrimaryColor from '../../app/assets/design-patterns/visualizations/use-primary-color.png';
 
 <!-- KEYWORDS: design pattern visualization map chart highcharts chartjs mapbox -->
 
 # Visualizations
 
 <ImageGrid regularWidth images={[VisualizationBanner]} />
```

Comment:
I like the right side of this, but the axis pieces make it feel like a lot of empty space.

This also violates our principle below about using the branding colors for charts 😆 , but I think it's okay to match the style of the other banners.

Reply:
The other banners feel like they have a little more 'depth' whereas this one feels very flat...can we do something with a background or a legend?